reference/customers.v3.yml

```diff
@@ -697,9 +713,121 @@ paths:
           headers: {}
       tags:
         - Customer Addresses
+  /customers/settings:
+    get:
+      tags:
+        - Customer Settings
+      description: Gets the customer settings.
+      operationId: CustomerSettingsGet
+      responses:
+        "200":
+          description: Customer settings are returned.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/responses/CustomerSettingsResponse"
```

Comment:
The `schema.$ref` can not point to a `response`; it must point to a `schema` @valentindellangela. See definitions for existing endpoints as an example.

Reply:
@aglensmith  thanks for pointing this out. I've updated it but now I'm getting even more issues unrelated to my `customers.v3.yml` file 🤔